Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -755,14 +755,11 @@ void RenderObject::addPDFURLRect(const PaintInfo& paintInfo, const LayoutPoint&
 
     if (urlRect.isEmpty())
         return;
-    RefPtr element = [this]() -> RefPtr<Element> {
-        RefPtr node = this->node();
-        if (!is<Element>(node) || !node->isLink())
-            return nullptr;
-        return downcast<Element>(WTFMove(node));
-    }();
-    if (!element)
+
+    RefPtr element = dynamicDowncast<Element>(this->node());
```

Comment:
Just `node()` is fine now. The `this->` was needed in the old code because the name `node` was also a local variable.